keyboards/dreamystate/dy16/dy16.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_ortho_5x4( \
```

Comment:
The image in the readme shows a 4x4 layout - where's this extra row coming from?

Reply:
In 'qmk_firmware\keyboards\tetris', the author used the encoder to correspond to the row5 keys. I'm just making space for future upgrades.

Reply:
This is what revision folders are for. If you're planning multiple revisions, you should make this board 4x4, and then place the configuration specific to this revision under `rev1`, etc.

Reply:
I see. When my code is finished, I will upload it directly. thank you.